data-protocols/dsp/dsp-version/dsp-version-http-api/src/main/java/org/eclipse/edc/protocol/dsp/version/http/api/DspVersionApiExtension.java

```diff
@@ -51,17 +56,26 @@ public class DspVersionApiExtension implements ServiceExtension {
     @Inject
     private VersionProtocolService service;
 
+    @Inject
+    private JsonLd jsonLd;
+    @Inject
+    private TypeManager typeManager;
+
+
     @Override
     public String name() {
         return NAME;
     }
 
     @Override
     public void initialize(ServiceExtensionContext context) {
+        var jsonLdMapper = typeManager.getMapper(JSON_LD);
+
         transformerRegistry.register(new JsonObjectFromProtocolVersionsTransformer());
         transformerRegistry.register(new JsonObjectFromVersionsError(Json.createBuilderFactory(Map.of())));
 
         webService.registerResource(ApiContext.PROTOCOL, new DspVersionApiController(requestHandler, service));
+        webService.registerDynamicResource(ApiContext.PROTOCOL, DspVersionApiController.class, new JerseyJsonLdInterceptor(jsonLd, jsonLdMapper, DSP_SCOPE_V_08));
```

Comment:
version endpoint has been introduced in version 2024/1 so it should have the related namespace (as pointed from the documentation: https://docs.internationaldataspaces.org/ids-knowledgebase/dataspace-protocol/common-functionalities/common.protocol)

not sure on what should happen with the next version tho...

Reply:
The version endpoint actually cannot have a version and it should change in the future i think.
 but we currently use 0.8 namespace for terms , that's why i binded it to 0.8

Reply:
If we are ok to do the "breaking change" we can do it once we introduce 2024/1 as namespace